### PR TITLE
Bumping version so we can distribute version of egg that has the new …

### PR DESCRIPTION
**Pull request by christopappas**
…convenience management commands for devs

### CODE REVIEW

**Review by brittneyexline — APPROVED** [review]

**Review by adamstankiewicz — APPROVED** [review]

File: CHANGELOG.rst [hunk 1]
```diff
@@ -14,6 +14,10 @@ Change Log
 Unreleased
 ----------
 
+[1.0.15] - 2019-01-24
+---------------------
+* Bumping version so others can install newer version of this app that includes convenient management commands for devs
```
Comment (irfanuddinahmad) [inline CHANGELOG.rst:19]: nit: * Includes `create_enterprise_user`, `create_enterprise_enrollment` management commands for creating demo test data for local development

**Review by irfanuddinahmad — COMMENTED** [review]
LGTM ... just a small nit in the CHANGELOG

File: CHANGELOG.rst (hunk 1 above)
Comment (christopappas) [reply CHANGELOG.rst:19]: added! good to be specific, i agree

**Comment by mushtaqak** [comment]
@christopappas Thanks for the commands ;)

**Pull request merged**